cli/pkg/kctrl/local/sources/template.go

```diff
@@ -1,50 +1,40 @@
 // Copyright 2022 VMware, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-package init
+package sources
 
 import (
+	"fmt"
 	"path/filepath"
 	"strings"
 
 	"github.com/cppforlife/go-cli-ui/ui"
 	cmdcore "github.com/vmware-tanzu/carvel-kapp-controller/cli/pkg/kctrl/cmd/core"
+	buildconfigs "github.com/vmware-tanzu/carvel-kapp-controller/cli/pkg/kctrl/local/buildconfigs"
 	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/apis/kappctrl/v1alpha1"
 )
 
-const (
-	UpstreamFolderName = "upstream"
-	StdIn              = "-"
-)
-
-type TemplateStep struct {
+type TemplateConfiguration struct {
```

Comment:
Should this just be `Template`? (`TemplateConfiguration.Configure` sounds a bit weird)

Reply:
Makes sense! I wonder if there could be a better name though

Reply:
Changed it to template for now